react-example/src/Components/LayoutLogin/LayoutLogin.less

```diff
@@ -0,0 +1,46 @@
+main{
+  display: grid;
+  width: 100vw;
+  grid-template-columns: 1fr 1fr;
+  height: 100vh;
+}
+
+.layout-login{
+  display: grid;
+  width: 100%;
+  background-color: #fff;
+  justify-content: center;
+  .div-login{
+    width: 350px;
+  }
+}
+
+.layout-img{
+  width: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.login-head{
+  width: 100%;
+  margin-top: 28vh;
+  p{
+    font-size: 1.6rem;
+    line-height: 1.9rem;
+    font-weight: 500;
+    color: #2D3748;
+    margin: 0;
+    padding: 0;
+  }
+  h1{
```

Comment:
Also leave a line break between the end of a selector and the beginning of the next one